src/Makefile.am

```diff
@@ -205,6 +205,7 @@ UNITE_CORE_H = \
   policy/policy.h \
   policy/rbf.h \
   proposer/block_builder.h \
+  proposer/finalization_reward_logic.h \
```

Comment:
We strive to keep this list alphabetically sorted.

Reply:
Fixed